### DISCUSSION TITLE
FR: track files in $HOME (mostly dot files)

### DISCUSSION THREAD

**Discussion by cig0** [jj-vcs/jj#3478] [Q&A] [answered]
Hi all :wave: 

**Describe the solution you'd like**
Has anyone considered using Jujutsu to monitor and backup changes to their $HOME dot files (And maybe other files as well)?

Despite I use Chezmoi extensively, Jujutsu's auto-commit feature is a killer one I'd like to add to my workflow.

Should I expect to encounter any issues besides taking care not to commit and push any sensitive information?

**Comment by khionu** [comment, marked as answer]
This wouldn't really be any different than using git to do so. jj won't auto-commit between running jj commands (maybe in the future if/when we have a daemon), and it won't ever auto-push, so there isn't really an advantage to jj over git for this specifically. I'm just reading up on chezmoi, so it's possible I'm missing something, but I'm pretty sure there's no catches here.

**Reply by cig0** [reply]
@khionu Thanks for sharing your thoughts!

Right. As you say, one could use Git to track stuff, but I want to use JJ because it is a massive improvement over plain Git.

**Reply by khionu** [reply]
Yeah, that should be no problem, same considerations as with git. Just note you will likely want to use `jj git init --colocate` to set it up, as that makes for the best integration with other tools expecting git.

**Reply by cig0** [reply]
Awesome. Thanks again for kindly stepping in, and for your suggestions :fist_oncoming:

**Reply by cig0** [reply]
I'm sorry for the necrobump. I want to confirm that the tandem Chezmoi + Jj is love written in the stars. I don't know how I've survived so far without Jj having my back :metal:
